specification/storage/resource-manager/Microsoft.Storage/stable/2019-06-01/queue.json

```diff
@@ -0,0 +1,570 @@
+            "$ref": "./storage.json#/parameters/StorageAccountName"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/QueueServicesName"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK -- returned the properties of a storage account’s Queue service.",
+            "schema": {
+              "$ref": "#/definitions/QueueServiceProperties"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Storage/storageAccounts/{accountName}/queueServices/default/queues/{queueName}": {
+      "put": {
+        "tags": [
+          "QueueService"
+        ],
+        "operationId": "PutQueue",
+        "x-ms-examples": {
+          "QueueOperationPut": {
+            "$ref": "./examples/QueueOperationPut.json"
+          },
+          "QueueOperationPutWithMetadata": {
+            "$ref": "./examples/QueueOperationPutWithMetadata.json"
+          }
+        },
+        "description": "Creates a new queue with the specified queue name, under the specified account.",
+        "parameters": [
+          {
+            "$ref": "./storage.json#/parameters/ResourceGroupName"
+          },
+          {
+            "$ref": "./storage.json#/parameters/StorageAccountName"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/QueueName"
+          },
+          {
+            "name": "queue",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/Queue"
+            },
+            "description": "Queue properties and metadata to be created with"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK -- Creates a queue with the given queue name successfully.",
+            "schema": {
+              "$ref": "#/definitions/Queue"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      },
+      "patch": {
```

Comment:
What can be patched? Only meta data?
It seems patch API is not so necessary.

Reply:
Yes for now its just the metadata, we will be including more properties and parameters as and when requested. We do the same for File Blobs as well.